build.gradle.kts

```diff
@@ -86,15 +86,16 @@ dependencies {
     implementation("net.logstash.logback:logstash-logback-encoder:6.3")
     implementation("com.microsoft.azure:applicationinsights-spring-boot-starter:2.5.1")
     implementation("com.microsoft.azure:applicationinsights-logging-logback:2.5.1")
-    implementation("com.github.timpeeters:spring-boot-graceful-shutdown:2.2.0")
+    implementation("com.github.timpeeters:spring-boot-graceful-shutdown:2.2.1")
     implementation("io.springfox:springfox-swagger2:2.9.2")
     implementation("io.springfox:springfox-swagger-ui:2.9.2")
     implementation("io.jsonwebtoken:jjwt:0.9.1")
     implementation("net.sf.ehcache:ehcache:2.10.6")
     implementation("org.apache.commons:commons-lang3:3.9")
-    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.10.2")
-    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
-    implementation("com.pauldijou:jwt-core_2.11:4.2.0")
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.10.3")
+    implementation( "com.fasterxml.jackson.module:jackson-module-kotlin:2.10.3")
+    implementation("com.pauldijou:jwt-core_2.11:4.3.0")
+    implementation("com.google.guava:guava:28.2-jre")
```

Comment:
Do you know how this has been added?

Reply:
A really old version is pulled in via springfox - this is manually overriding that version.

Its a little bit scary as guava arent afraid of breaking backwards compatibility but other services use the same version of springfox and this version of guava so imagine itll be alright.